wpt-status/index.html

```diff
@@ -136,27 +141,141 @@
     margin-left: 1.5em;
     margin-top: 0.2em;
 }
+
+
+#controls {
+    display: flex;
+    align-items: center;
+    gap: 0.5rem;
+    flex-wrap: nowrap;
+    flex-shrink: 0;
+}
+
+.view-toggle {
+    display: inline-flex;
+    border: 1px solid #ccc;
+    border-radius: 6px;
+    overflow: hidden;
+    box-shadow: 0 1px 2px rgba(0, 0, 0, 0.1);
+}
+
+.toggle-btn {
+    appearance: none;
+    border: none;
+    background: #f5f5f5;
+    padding: 0.25em 0.6em;
+    font-size: 0.9em;
+    cursor: pointer;
+    color: #333;
+    transition: background 0.2s, color 0.2s;
+}
+
+.toggle-btn:hover {
+    background: #eaeaea;
+}
+
+.toggle-btn.active {
+    background: #0066cc;
+    color: white;
+}
+
+#controls label,
+#controls span {
+    white-space: nowrap;
+}
+
+.info-run {
+    flex: 0 1 auto;
+    min-width: 0;
+    font-family: monospace;
+    background-color: #f5f5f5;
+    color: #333;
+    padding: 0.3em 0.6em;
+    border: 1px solid #ccc;
+    border-radius: 4px;
+    display: inline-block;
+    white-space: nowrap;
+    font-size: 0.9em;
+    overflow: hidden;
+    text-overflow: ellipsis;
+    text-align: right;
+}
+
+.tight-label {
+    margin-right: -0.5em; /* pulls WPE closer to the select */
+}
+
+.filter-input-notfound {
+    background-color: #62293c;
+    color: white;
+}
+
 </style>
 
 <div id="header">
-  <h1>WPE Specification Support</h1>
+  <h1>WPE Web Support Status</h1>
 </div>
 
 <div id="stickyheader">
-<span><label>WPE version:<select id="version-select"></select><strong> <span id="meta-browser-version"></span></strong></label></span><span>Results from <strong><a id="meta-wpt-runlink" href="" target="_blank"></a></strong> (Date: <strong><span id="meta-run-date"></span></strong>)</span>
+  <div id="controls">
+    <span title="Result source">&#x1F4CA;</span>
+
+    <div class="view-toggle">
+      <button class="toggle-btn" data-view="BCD" title="BCD Results" >BCD</button>
```

Comment:
I guess realistically these are trying to be tabs but they aren't actually implemented as tabs. Probably I'd just find a good tabs custom element and use it here..

Reply:
Good suggestion. I have just pushed a commit changing the approach to use tabs.